packages/token-search-discovery-controller/src/types.ts

```diff
@@ -15,6 +15,8 @@ export type TopLosersParams = ParamsBase;
 
 export type TopGainersParams = ParamsBase;
 
+export type BlueChipParams = ParamsBase;
```

Comment:
In order to access this type on the mobile app you'll want to export it from `/token-search-discovery-controller/src/index.ts` too.  If not they'll have to access the .jsc file.